packages/gatsby-source-wordpress/src/steps/process-and-validate-plugin-options.ts

```diff
@@ -46,6 +47,51 @@ const optionsProcessors: Array<IOptionsProcessor> = [
 
       delete userPluginOptions.schema.queryDepth
 
+      return userPluginOptions
+    },
+  },
+  {
+    name: `Require beforeChangeNode type setting functions by absolute or relative path`,
+    test: ({ userPluginOptions }: IProcessorOptions): boolean =>
+      !!userPluginOptions?.type,
+    processor: ({
+      helpers,
+      userPluginOptions,
+    }: IProcessorOptions): IPluginOptions => {
+      const gatsbyStore = helpers.store.getState()
+      const typeSettings = Object.entries(userPluginOptions.type)
+
+      typeSettings.forEach(([typeName, settings]) => {
+        const beforeChangeNodePath = settings?.beforeChangeNode
+
+        if (!beforeChangeNodePath || typeof beforeChangeNodePath !== `string`) {
+          return
+        }
+
+        try {
+          const absoluteRequirePath: string | undefined = path.isAbsolute(
+            beforeChangeNodePath
+          )
+            ? beforeChangeNodePath
+            : require.resolve(
```

Comment:
@wardpeet you mentioned gatsby-fs. Is that available now or will this need to be updated in the future?